relay-server/server/relayServer.go

```diff
@@ -116,7 +116,7 @@ func (ls *LogService) removeMsgStruct(uid string) {
 // WatchMessages Function
 func (ls *LogService) WatchMessages(req *pb.RequestMessage, svr pb.LogService_WatchMessagesServer) error {
 	uid := uuid.Must(uuid.NewRandom()).String()
-	conn := make(chan *pb.Message, 1)
+	conn := make(chan *pb.Message, 1000)
```

Comment:
What's the reasoning/impact for this?

Reply:
while testing didnt see alot of message being sent, so it doesnt need the same buffer size as log